app/src/main/java/com/secuso/privacyfriendlycodescanner/qrscanner/ui/resultfragments/URLResultFragment.java

```diff
@@ -92,7 +92,8 @@ public void onProceedPressed(Context context) {
         } else {
             String caption = "";
             String qrurl3="";
-            if(!qrurl.startsWith("http://") && !qrurl.startsWith("https://"))
+            final String lowercase_qrurl = qrurl.toLowerCase();
```

Comment:
I would probably have used a case insensitive pattern to match for http(s)://
But this will do.. thank you